src/api/Item.js

```diff
@@ -329,14 +335,14 @@ class Item extends Base {
         }
 
         try {
-            const { id, permissions }: BoxItem = item;
+            const { id, permissions, shared_link: sharedLink }: BoxItem = item;
             this.id = id;
             this.successCallback = successCallback;
             this.errorCallback = errorCallback;
 
-            // if we use the default access level, we don't need permission to set the access level
-            const canSkipSetShareAccess = access === undefined;
-            this.validateRequest(id, permissions, canSkipSetShareAccess);
+            // skip permission check when creating links with default access level
+            const canSkipSetShareAccessPermission = !sharedLink && access === undefined;
```

Comment:
we only want to skip the permission check if the shared link does not already exist. i.e. the scenario in which a "viewer" role is trying to create a shared link